src/test/java/uk/gov/hmcts/juror/api/moj/service/report/YieldPerformanceReportServiceImplTest.java

```diff
@@ -0,0 +1,196 @@
+package uk.gov.hmcts.juror.api.moj.service.report;
+
+import lombok.SneakyThrows;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.juror.api.TestUtils;
+import uk.gov.hmcts.juror.api.moj.controller.reports.request.YieldPerformanceReportRequest;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.AbstractReportResponse;
+import uk.gov.hmcts.juror.api.moj.controller.reports.response.YieldPerformanceReportResponse;
+import uk.gov.hmcts.juror.api.moj.exception.MojException;
+import uk.gov.hmcts.juror.api.moj.repository.CourtQueriesRepository;
+import uk.gov.hmcts.juror.api.moj.repository.IPoolCommentRepository;
+import uk.gov.hmcts.juror.api.moj.repository.JurorPoolRepository;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.BDDAssertions.within;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.juror.api.moj.controller.reports.response.YieldPerformanceReportResponse.TableHeading.BALANCE;
+import static uk.gov.hmcts.juror.api.moj.controller.reports.response.YieldPerformanceReportResponse.TableHeading.COMMENTS;
+import static uk.gov.hmcts.juror.api.moj.controller.reports.response.YieldPerformanceReportResponse.TableHeading.CONFIRMED;
+import static uk.gov.hmcts.juror.api.moj.controller.reports.response.YieldPerformanceReportResponse.TableHeading.COURT;
+import static uk.gov.hmcts.juror.api.moj.controller.reports.response.YieldPerformanceReportResponse.TableHeading.DIFFERENCE;
+import static uk.gov.hmcts.juror.api.moj.controller.reports.response.YieldPerformanceReportResponse.TableHeading.REQUESTED;
+
+@SuppressWarnings({
+    "PMD.AssertionsShouldIncludeMessage",
+    "PMD.UnnecessaryFullyQualifiedName"
```

Comment:
"PMD.UnnecessaryFullyQualifiedName" should not be suppressed.

"PMD.AssertionsShouldIncludeMessage", should be used on method level in this context

Reply:
removed these as no longer needed